ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1608,3 +1608,149 @@
+              name: CONTROLPLANE_NODE_ENI
+              trim: 'true'
+          - string:
+              default: ''
+              description: The ec2 secondary network-interface id of worker Node.
+              name: WORKER_NODE_ENI
+              trim: 'true'
+          - string:
+              default: ''
+              description: The subnet CIDR reservation id created in the subnet during test.
+              name: SUBNET_CIDR_RESERVATION_ID
+              trim: 'true'
+          builders:
+          - shell: |-
+              #!/bin/bash
+              set -ex
+              export CONTROLPLANE_INSTANCE_ID=${{CONTROLPLANE_INSTANCE_ID}}
+              export WORKER_INSTANCE_ID=${{WORKER_INSTANCE_ID}}
+              export CONTROLPLANE_NODE_ENI=${{CONTROLPLANE_NODE_ENI}}
+              export WORKER_NODE_ENI=${{WORKER_NODE_ENI}}
+              export SUBNET_CIDR_RES_ID=${{SUBNET_CIDR_RESERVATION_ID}}
+              source ./ci/test-sriov-secondary-network-aws.sh --cleanup-only \
+                --aws-access-key ${{AWS_ACCESS_KEY}} --aws-secret-key ${{AWS_SECRET_KEY}} --aws-service-user-role-arn ${{AWS_SERVICE_USER_ROLE_ARN}} --aws-service-user ${{AWS_SERVICE_USER_NAME}} \
+                --CONTROLPLANE_INSTANCE_ID ${{CONTROLPLANE_INSTANCE_ID}}
+          concurrent: false
+          disabled: false
+          node: antrea-cloud
+          branches:
+          - '${{ANTREA_GIT_REVISION}}'
+          repo_url: '${{ANTREA_REPO}}'
+          publishers:
+          - email:
+              notify-every-unstable-build: true
+              recipients: projectantrea-dev@googlegroups.com rahulj@vmware.com
+          triggers: []
+          wrappers:
+          - credentials-binding:
+            - text:
+                credential-id: AWS_ACCESS_KEY # Jenkins secret that stores aws access key
+                variable: AWS_ACCESS_KEY
+            - text:
+                credential-id: AWS_SECRET_KEY # Jenkins secret that stores aws secret key
+                variable: AWS_SECRET_KEY
+            - text:
+                credential-id: AWS_SERVICE_USER_ROLE_ARN
+                variable: AWS_SERVICE_USER_ROLE_ARN
+            - text:
+               credential-id: AWS_SERVICE_USER_NAME
+               variable: AWS_SERVICE_USER_NAME
+      - '{name}-{test_name}-for-pull-request':
+          disabled: false
+          test_name: sriov-secondary-network-e2e
+          node: antrea-cloud-test
+          description: 'Test SR-IOV secondary network in AWS for antrea.'
+          branches:
+          - ${{sha1}}
+          trigger_phrase: '^(?!Thanks for your PR).*/test-sriov-secondary-network-e2e.*'
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
```

Comment:
`trigger_permit_all: true` means all users can trigger this job.

Reply:
Thanks. Updated.